### PR TITLE
TensorBoard loss visualisation

### PR DESCRIPTION
**Pull request by matteo-dunnhofer**
I added the tf.summary instructions to visualise the loss of both yolo and yolov2. For the loss_mva, I need some days to figure out how treat it like a graph operation (at this time it is not). I'll hope to have it done by next week.

To visualise the loss in TensorBoard, just type `tensorboard --logdir="summary"` in your terminal.

Hope everything is good ✌️

### CODE REVIEW

**Comment by 1NNcoder** [comment]
I just tried this works well, any way we can get the graph section working with ` add_graph()`?

File: net/build.py [hunk 1]
```diff
@@ -104,6 +104,10 @@ def setup_meta_ops(self):
 			cfg['device_count'] = {'GPU': 0}
 
 		if self.FLAGS.train: self.build_train_op()
+
+		self.summary_op = tf.summary.merge_all()
+		self.writer = tf.summary.FileWriter(self.FLAGS.summary + 'train')
```
Comment (thtrieu) [inline net/build.py]: I suggest using `os.path.join()`.

**Comment by dontfollowmeimcrazy** [comment]
@1NNcoder Just forgot to uncomment this line https://github.com/thtrieu/darkflow/blob/bb677792d47026c4fc6d67a6f5a9295b941197cf/net/build.py#L119 . Uncomment it and you can visualise the graph.

**Comment by Alemolet** [comment]
It doesn't work for me if I write tensorboard --logdir="summary" after the training; when I open localhost:6006 on Chrome it says ''no dashboards are active for the current dataset"

**Comment by mfaramarzi** [comment]
add --summary summary in your training command e.g.

flow --model cfg/tiny-yolo-voc-3c.cfg --load -1 --train --annotation Annotations --dataset Images --gpu 0.9 --lr .0003 --trainer adam --batch 128 --labels labels-RDD.txt --summary summary

**Comment by ismaelfer** [comment]
Thanks!!

**Pull request merged**